test/e2e/checkpoint_test.go

```diff
@@ -232,6 +232,7 @@ var _ = Describe("Podman checkpoint", func() {
 	})
 
 	It("podman checkpoint container with established tcp connections", func() {
+		Skip("fails on ubuntu")
```

Comment:
Why is this a plain Skip? Shouldn't this be a `SkipIfNotFedora()` too?

Reply:
Inherited from #6418 but a good catch! Will update with the next push.